WordPress/openverse
# Update Django admin views for bulk moderation ergonomics

## Fixes



Fixes #3842 by @stacimc 
Fixes #3843 by @stacimc
Fixes #3844 by @stacimc 

## Description



1. This PR adds a filter by decision ID to the sidebar of the sensitive media and deleted media list views. Using this filter the list can be quickly narrowed down to all the meida affected by one decision, making it easier to operate on them again, as a bulk.

   ![image](https://github.com/user-attachments/assets/8fd465f9-a3cb-4da6-a84a-06ed2ec56ac9)

2. This PR also updates the decision list view to add a single/bulk decision filter.

   ![image](https://github.com/user-attachments/assets/393d3776-1f80-467d-9db2-f6937f12d51a)

3. Finally the PR links to the sensitive media or deleted media list views from the single decision page, automatically populating the filter parameter described in point 1 above.

  <img width="1392" alt="Screenshot 2024-07-30 at 9 54 55 PM" src="https://github.com/user-attachments/assets/714c6dcc-e873-437a-9e76-e31b3711288d">


## Testing Instructions


0. Check out the PR and run the dev server.
1. Create some reports and take decisions on them from the single media view or take decisions on media items directly using bulk moderation actions from the media list view.
2. Go to the sensitive media list and filter by a decision ID. You should only UUIDs for  media that were affected by that decision.
3. Go to the deleted media list and filter by a decision ID. You should only see UUIDs for media that were affected by that decision.
4. Go the decision list view. Filter the decisions by single or bulk. Based on your actions in step 1, you will see a subset of decisions.
5. Click on a deindexing decision, you will see a "View deleted" button in top right, clicking which takes you to the pre-filtered deleted media list view.
6. Click on a sensitive marking decision, you will see a "View sensitive" button in the top right, clicking which tkaes you to the pre-filtered sensitive media list view.
 
## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
